src/pages/Validators/Stats/ActiveValidators.tsx

```diff
@@ -1,37 +1,26 @@
 // Copyright 2022 @paritytech/polkadot-staking-dashboard authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import BN from 'bn.js';
 import { useStaking } from 'contexts/Staking';
 import { Pie } from 'library/StatBoxList/Pie';
 import { useTranslation } from 'react-i18next';
-import { toFixedIfNecessary } from 'Utils';
 
 const ActiveValidatorsStatBox = () => {
-  const { staking, eraStakers } = useStaking();
-  const { validatorCount } = staking;
+  const { eraStakers } = useStaking();
   const { activeValidators } = eraStakers;
   const { t } = useTranslation('pages');
 
-  // active validators as percent
-  let activeValidatorsAsPercent = 0;
-  if (validatorCount.gt(new BN(0))) {
-    activeValidatorsAsPercent =
-      activeValidators / (validatorCount.toNumber() * 0.01);
-  }
-
   const params = {
     label: t('validators.active_validators'),
     stat: {
       value: activeValidators,
-      total: validatorCount.toNumber(),
```

Comment:
I think we still have max # of validators.

Reply:
Yup, it's shown in the box on the left. This one just will show the active validators in current era
<img width="593" alt="image" src="https://user-images.githubusercontent.com/7670051/205683750-0a409af4-83a9-44f1-9b18-8272e4b6026a.png">

Reply:
Awesome.